flask/app/routes.py

```diff
@@ -162,3 +163,42 @@ def change_password():
     except:
         db.session.rollback()
         return 'Server error', 500
+
+
+@app.route('/api/participants/<participant_id>', methods = ['PATCH'])
+# @login_required # check document for how to test endpoints with login signing? 
+def update_participants(participant_id):
+    patch_json = request.get_json()
+
+    # convert to dictionary
+    patch_d = json.loads(patch_json)
```

Comment:
`request.get_json()` already returns dict or None. Check for `None` and return a 400 Bad Request.